candig/server/serverconfig.py

```diff
@@ -194,8 +194,18 @@ class TykConfig(KeycloakOidConfig):
     Configuration to use when forwarding requests through the API gateway.
     This also requires that keycloak config is being used and is set up properly.
 
+    If OPA_SERVER is set, it will be used as source of access_map instead of ACCESS_LIST.
+    OPA_SERVER link needs to be the OPA's address plus the '/data/permissions/datasets'
```

Comment:
Is it plus `/v1/data/permissions/datasets` or `/data/permissions/datasets`?

Reply:
My bad, it is plus /v1/data/permissions/datasets